README.md

```diff
@@ -10,63 +10,45 @@ Welcome to the official repository for the Super Mario Odyssey Online mod! Have
-4. Launch the game! Upon first time bootup, the mod should ask for a server IP to save to the games common save file. This IP address will be the server you wish to connect to every time you launch the game with the mod installed. (Note: un-installing the mod and launching the game will remove the server IP from the common save file.)
+  - [devkitPro](https://devkitpro.org/) 
+  - Python 3
+  - The [Keystone-Engine](https://www.keystone-engine.org/) Python Module
 
-## Gamemode Info
-### Hide and Seek
-* Depending on Group size, select who will start as seekers at the beginning of each round and a kingdom to hide in. 
-* Each player has a timer on the top right of the screen that will increase while they are hiding during a round. 
-* When a seeker gets close enough to a player, the player will die and respawn as a seeker.
-* During the round, hiders who die by other means will also become seekers upon respawning.
-* If a hider loads into a new stage (via a pipe, door, etc.) the hider will get 5 seconds of tag invincibility to prevent spawn point camping.
-* The player with the most time at the end of a round (or set of rounds) is considered the winner.
-* While not a concrete rule, it's generally agreed upon that hiding should not be done out of bounds, inside objects that don't sync across games yet, and inside objects that completely conceal a player from others (such as trees).
+  ### Building
 
-## Gamemode Controls
-### Hide and Seek
-- Left D-Pad: Decrease time
-- Right D-Pad: Increase Time
-- L + D-Pad Down: Reset Time
-- D-Pad Up: Switch from Hider/Seeker
+  Build has only been tested on WSL2 running Ubuntu 20.04.1.
 
-## Building Prerequisites
+  Just run:
+  ```
+  DEVKITPRO={path_to_devkitpro} make
+  ```
 
-- [devkitPro](https://devkitpro.org/) 
-- Python 3
-- The [Keystone-Engine](https://www.keystone-engine.org/) Python Module
+  On Ubuntu (and other Debian-based systems), devkitPro will be installed to `/opt/devkitpro` by default:
 
-## Building
+  ```
+  DEVKITPRO=/opt/devkitpro/ make
+  ```
 
-Build has only been tested on WSL2 running Ubuntu 20.04.1.
+  ### Installing (Atmosphère)
 
-Just run:
-```
-DEVKITPRO={path_to_devkitpro} make
-```
+  After a successful build, simply transfer the `atmosphere` folder located inside `starlight_patch_100` to the root of your switch's SD card.
+</details>
 
-On Ubuntu (and other Debian-based systems), devkitPro will be installed to `/opt/devkitpro` by default:
+## Troubleshooting
 
-```
-DEVKITPRO=/opt/devkitpro/ make
-```
-
-## Installing (Atmosphère)
-
-After a successful build, simply transfer the `atmosphere` folder located inside `starlight_patch_100` to the root of your switch's SD card.
+The [Super Mario Odyssey website](https://smoo.it) has a FAQ section that should solve many issues.
+However, for any further questions or help not covered by the site, please visit the [CraftyBoss Community Discord Server](discord.gg/jYCueK2BqD) and ask in the `help`/`help-2` channel. 
```

Comment:
Where is this Discord invite link coming from?
(Not that it expires soon or something.)

On the website I'm using https://discord.gg/w3TnB899ww which is from the video description of this YouTube video by CraftyBoss: https://www.youtube.com/watch?v=nXxnhvP_LEI

Reply:
That was the link in the repo description. I can switch it if you prefer

Reply:
No need to then, it can stay as it is.

Reply:
Works for me.